android/src/main/java/com/stripeterminalreactnative/StripeTerminalReactNativeModule.kt

```diff
@@ -1082,6 +1082,51 @@ class StripeTerminalReactNativeModule(reactContext: ReactApplicationContext) :
         promise.resolve(mapFromReaderSupportResult(readerSupportResult))
     }
 
+    @ReactMethod
+    @Suppress("unused")
+    fun setLocalMobileUxConfiguration(params: ReadableMap, promise: Promise) = withExceptionResolver(promise) {
+        val localMobileUxConfigurationBuilder = LocalMobileUxConfiguration.Builder()
+
+        var tapZone: LocalMobileUxConfiguration.TapZone? = null
+        val tapZoneParam = params.getMap("tapZone")
+        tapZoneParam?.let {
+            val tapZoneIndicator = mapToTapZoneIndicator(tapZoneParam.getString("tapZoneIndicator"))
+
+            val tapZonePosition = tapZoneParam.getMap("tapZonePosition")?.let {
+                LocalMobileUxConfiguration.TapZonePosition.Manual(
+                    it.getDouble("xBias").toFloat(),
+                    it.getDouble("yBias").toFloat())
+            }
+
+            val tapZoneBuilder = LocalMobileUxConfiguration.TapZone.Manual.Builder()
+            tapZoneIndicator.let { tapZoneBuilder.indicator(it) }
+            tapZonePosition?.let { tapZoneBuilder.position(it) }
+            tapZone = tapZoneBuilder.build()
+        }
+        localMobileUxConfigurationBuilder.tapZone(tapZone?:LocalMobileUxConfiguration.TapZone.Default)
+
+        val colorsParam = params.getMap("colors")
+        colorsParam?.let {
+            val colorSchemeBuilder = LocalMobileUxConfiguration.ColorScheme.Builder()
+            colorSchemeBuilder.apply {
+                primary(it.getString("primary")?.let {
+                    LocalMobileUxConfiguration.Color.Value(hexToArgb(it)) } ?:
+                    LocalMobileUxConfiguration.Color.Default)
```

Comment:
nit: you can probably pull this out into a private extension method for re-use

```kotlin
private fun String?.toLocalMobileColor(): LocalMobileUxConfiguration.Color {
    return this
        ?.let { LocalMobileUxConfiguration.Color.Value(hexToArgb(it)) }
        ?: LocalMobileUxConfiguration.Color.Default
}
```

And then call it like:

```kotlin
colorSchemeBuilder.apply {
    primary(it.getString("primary").toLocalMobileColor())
    success(it.getString("success").toLocalMobileColor())
    error(it.getString("error").toLocalMobileColor())
}
```